### ISSUE TITLE
Main menu theme is offtopic

### ISSUE THREAD

**Issue by ghost** [fix/Lisk-Lite-Client#31]
<img width="390" alt="screen shot 2016-08-11 at 11 31 09" src="https://cloud.githubusercontent.com/assets/9548618/17584537/4f7c770e-5fb7-11e6-8ea9-356c3abc8fd0.png">
I can not understand why there is cannon on background? This is totally off-topic with lisk. I suggest redesign.

**Comment by fix** [comment]
yes just the first image that was flat enough to lay the information on it.

**Comment by fix** [comment]
if you can suggest some, it needs to be:
- quite flat (not many high frequencies)
- dark on the top (to have clear menu that are white font)
- license under cc

**Comment by ghost** [comment]
In OSX/iOS there is effect called UIVisualEffectView with pretty fancy look which can be solution.
<img width="830" alt="screen shot 2016-08-11 at 17 05 29" src="https://cloud.githubusercontent.com/assets/9548618/17593351/e08b857c-5fe5-11e6-820c-2988c852a295.png">
Not sure if there is similar library in js but should be something.

**Comment by fix** [comment]
I have changed it to https://www.pexels.com/photo/flight-nature-sky-flying-36487/

**Comment by ghost** [comment]
For me both background are just fine but it's still off-topic, just my personal point of view. Maybe the best background will be no background ?
